tests/PKSim.Tests/Core/ClassificationMapperSpecs.cs

```diff
@@ -37,11 +37,10 @@ public class When_mapping_snapshot_to_classification : concern_for_Classificatio
       private OSPSuite.Core.Domain.Classification _result;
       private Classification _child;
 
-      protected override Task Context()
+      protected override async Task Context()
       {
-         base.Context();
```

Comment:
I did not catch that before sorry . Since the base.Context is a Task, it needs to be awaited. Otherwise the next line could be executed BEFORE the base.Context is finished (at least in theory depending on what the base class is doing)